cypress/integration/models/administrator/credentials/credentials.ts

```diff
@@ -0,0 +1,63 @@
+import {
+    applyAction,
+    click,
+    clickByText,
+    inputText,
+    selectUserPerspective,
+    selectWithin,
+} from "../../../../utils/utils";
+import { dropdownMenuToggle } from "../../../views/tags.view";
+import { administrator, button, credentials, deleteAction } from "../../../types/constants";
+import {
+    closeNotification,
+    confirmButton,
+    createBtn,
+    credentialNameInput,
+    descriptionInput,
+    modal,
+    navLink,
+} from "../../../views/credentials.view";
+
+export class Credentials {
+    name = "";
+    description = "";
+
+    constructor(name) {
+        this.name = name;
+    }
+
+    protected fillName(): void {
+        inputText(credentialNameInput, this.name);
+    }
+
+    protected fillDescription(): void {
+        if (this.description != "") {
+            inputText(descriptionInput, this.description);
+        }
+    }
+
+    protected selectType(type): void {
+        selectWithin(modal, dropdownMenuToggle);
+        clickByText(button, type);
+    }
+
+    static open() {
```

Comment:
Please rename it to clickCredentials() or openCredentials() in case this method needs to be used somewhere else.

Reply:
Where else it should be used? It is static method of credentials and it is not generic, it can't be used somewhere else.

Reply:
Renamed to `openList()`

Reply:
Example : 
https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/integration/tests/developer/applicationinventory/application/create.test.ts#L81
Please rename .